src/commands/context/selectDockerContext.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ProgressLocation, window } from 'vscode';
+import { ext } from '../../extensionVariables';
+import { localize } from '../../localize';
+import { dockerContextManager, IDockerContextListItem } from '../../utils/dockerContextManager';
+
+export async function selectDockerContext(prompt: string): Promise<IDockerContextListItem> {
+    const fetchingContexts = localize('vscode-docker.commands.context.fetchingContexts', 'Fetching Docker contexts...');
+    let contexts: IDockerContextListItem[];
+    await window.withProgress(
+        { location: ProgressLocation.Notification, title: fetchingContexts },
+        async () => {
+            contexts = await dockerContextManager.listAll()
+        });
+
+    const contextItems = contexts.map(ctx => ({
+        label: `${ctx.Name} ${ctx.Current ? '*' : ''}`.trim(),
```

Comment:
I'd do:
```typescript
    label: ctx.Name,
    detail: ctx.Current ? localize(..., 'Currently selected') : '',
```

You can also try `description` instead of `detail`, I'm not sure exactly how each is rendered--worth looking at both.

Reply:
This mimics the output of `docker context ls`, that is why I did it this way.

Reply:
Can we do the `*` as `detail` or `description`? @BigMorty what do you think?

Reply:
I like the more "(current)" or "(selected)" better than just an asterisk.

Reply:
OK will use `(current)`